init.el

```diff
@@ -58,7 +58,12 @@
   "The home of Prelude's core functionality.")
 (defvar prelude-modules-dir (expand-file-name  "modules" prelude-dir)
   "This directory houses all of the built-in Prelude modules.")
-(defvar prelude-personal-dir (expand-file-name "personal" prelude-dir)
+(defvar prelude-personal-dir (or (let ((personal-dir (getenv "PRELUDE_PERSONAL_DIR")))
+                                   (when personal-dir
+                                     (let ((personal-dir-exp (expand-file-name personal-dir)))
+                                       (when (file-directory-p personal-dir-exp)
```

Comment:
Seems to me without some feedback for the user that they set the wrong thing in the var, this check is kind of pointless.